recipes/flatbuffers/all/conanfile.py

```diff
@@ -132,7 +132,10 @@ def _module_path(self):
     def package_info(self):
         self.cpp_info.set_property("cmake_find_mode", "both")
         self.cpp_info.set_property("cmake_module_file_name", "FlatBuffers")
-        self.cpp_info.set_property("cmake_file_name", "Flatbuffers")
+        if Version(self.version) < "2.0.7":
+            self.cpp_info.set_property("cmake_file_name", "Flatbuffers")
+        else:
+            self.cpp_info.set_property("cmake_file_name", "FlatBuffers")
```

Comment:
See https://github.com/google/flatbuffers/pull/7378.

Reply:
prefer https://github.com/google/flatbuffers/pull/7502

Reply:
I changed it in d4c03f3a3267b27df147488e0878fab136f76c06.